.github/workflows/pr-commit.yaml

```diff
@@ -153,8 +153,32 @@ jobs:
       - uses: pnpm/action-setup@v2.0.1
         with:
           version: 6.0.2
+      - name: Set up Python
```

Comment:
This is just for testing during the migration, right?

Reply:
Yes. Currently, we still need Py deps to generate schemas. It will be removed after all migration is done.